charts/argo-templates/templates/diff-bq-table.yaml

```diff
@@ -137,6 +137,29 @@ spec:
           - name: rows-to-append-count
             valueFrom:
               parameter: '{{ "{{tasks.query-row-count.outputs.result}}" }}'
+      metrics:
+        prometheus:
+          - name: success-counter
```

Comment:
I worry about hard-coding things like this in `argo-templates`, as it could make it tough to tease apart metrics from different workflows (i.e. ClinVar vs. ENCODE).

For things that might make sense to embed within specific workflow templates (like row counts here), could we tag them with labels to distinguish between different ingest projects?

For things that feel like workflow-level metrics (i.e. this success-counter), can we instead define the YAML structure as a helper in `_helpers.tpl`, so we can inject them as needed across our workflows w/o duplicating code?

Reply:
Can we include a block for [realtime workflow duration](https://github.com/argoproj/argo/blob/master/docs/metrics.md#realtime-metrics) within the helpers? That'll be a cool one to track.